api/v1/numaresourcesscheduler_normalize.go

```diff
@@ -16,9 +16,39 @@
 
 package v1
 
+import (
+	"k8s.io/klog/v2"
+	"k8s.io/utils/ptr"
+	
+	"github.com/k8stopologyawareschedwg/deployer/pkg/deployer/platform"
+)
+
+const (
+	// ActivePodsResourcesSupportSince defines the OCP version which started to support the fixed kubelet
+	// in which the PodResourcesAPI lists the active pods by default
+	ActivePodsResourcesSupportSince = "4.20.999"
```

Comment:
this must be adjusted once we know since which OCP version is the kubelet fix supported

Reply:
correct. This is also in implementation detail, so the constant should start with a lowecase letter.